backend/tests/Integration/Request/RequestWriteSlicesTest.php

```
@@ -391,12 +448,28 @@ public function testSuspendAndResumeTransitionStatus(): void
 
         $resumed = $this->lifecycle($requestId, $suspended['lockVersion'], $manager, RequestAction::Resume);
         self::assertSame('in_progress', $resumed['status']);
+        self::assertNull((new RequestQuery($this->db()))->findDetails($requestId, $initiator)['item']['status_reason']);
+
+        $resuspended = $this->lifecycle($requestId, $resumed['lockVersion'], $manager, RequestAction::Suspend, 'Ожидание нового стенда');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$resuspended'. (undefined)

(UnusedLocalVariable)